spec/models/firm_spec.rb

```diff
@@ -461,18 +461,8 @@
       it 'cascades destroy to advisers' do
         adviser = firm.advisers.first
         firm.destroy
-        firm.run_callbacks(:commit)
-        adviser.run_callbacks(:commit)
         expect(Adviser.where(id: adviser.id)).to be_empty
```

Comment:
I believe so. In this test we're not testing side-effects from after_commit hooks we're testing just the cascading delete. The after_commit hooks are tested on line 515. Unless there's something we've missed?